cliboa/template/Pipfile.above36

```diff
@@ -38,7 +38,7 @@ azure-storage-blob = "==12.9.0"
 python-gnupg = "==0.4.8"
 openpyxl = "==3.0.9"
 pyminizip = "==0.2.5"
-psycopg2 = "==2.9.1"
+psycopg2-binary = "==2.9.1"
```

Comment:
document uses pip install to specify psycopg2-binary and uses the same calling as the implementation now merged.